### PR TITLE
[Accessibility] Improved contrast

### PR DESCRIPTION
**Pull request by ghost**
According to accessibility guidelines, the contrast ratio of text vs background should be at least 4.5 (7.0 is better!).

A lot of text in Isis (Bootstrap 2) fails this ratio.

This PR fixes a lot of these by tweaking either the font colour or the background colour of elements.
- Darker links
- Darker text/links in status bar (bottom)
- Darker headings
- Darker text in alerts
- Darker background of gray badges
- Darker background of button states (green/red)

contrast

### CODE REVIEW

**Comment by dgt41** [comment]
@test OK
Although this improves the contrast, I think a better approach will be to have a dedicated isis style e.g. `High Contrast ` where no element will have a value less than 10. This can be implemented in a similar way as tinyMCE is picking up templates (folders inside /css/ will define styles that can be selected through a dropdown menu in Template Manager: Edit Style)

screen shot 2015-03-17 at 4 17 55

**Comment by nonumber** [comment]
These tweaks are so minor, that most people will probably not even notice it.
That doesn't warrant a new style/template for me.
To have an extra style in isis with even more contrast (maybe even monochrome!) would be fine by me. But that would be a bigger step than these quick fixes that benefit all.

**Comment by pbwebdev** [comment]
Thanks Peter!

**Comment by n9iels** [comment]
@test looks good to me<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6475">issues.joomla.org/joomla-cms/6475</a>.</sub>

**Comment by nonumber** [comment]
@dgt41 I like the idea to have an extra style in isis that has a minimum contrast ratio of 7 (Which is WCAG20 level AAA).
But let's try to get isis at least to WCAG20 level AA, so a a minimum contrast ratio of 4.5.

For more info: http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

PS: I know contrast ratio is only 1 little part of the WCAG20... small steps.

**Comment by dgt41** [comment]
@nonumber I am not against this PR! Don’t take me wrong, all I proposed was the option for one or maybe more accessible themes within isis. One great starting point for the colors i guess can be http://contrastrebellion.com
Also this one got already 3 successful tests and it can be marked RTC and maybe merged for 3.4.2 (nothing major here to postpone it for another release)

**Comment by pbwebdev** [comment]
I do like that idea @dgt41 of having that extra high contrast level. Perhaps it can be added as a template option to switch over to the extra high level.

But yeh in this case as long as we pass WCAG 2.0 AA level in testing for the standard admin for everyone else is best!

Thanks everyone.

**Comment by zero-24** [comment]
Moving to RTC based on testing. Other changes can be done with a later PR. Thanks :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6475">issues.joomla.org/joomla-cms/6475</a>.</sub>

**Comment by wilsonge** [comment]
@nonumber can you fix conflicts so i can merge please :)

**Comment by nonumber** [comment]
Ok, done :)

File: administrator/templates/isis/less/template.less [hunk 1]
```diff
@@ -130,6 +129,10 @@ a:focus {
 	}
 }
 
+.navbar-inverse & {
```
Comment (wilsonge) [inline administrator/templates/isis/less/template.less]: The ```&``` symbol here seems wrong?

Comment (nonumber) [inline administrator/templates/isis/less/template.less]: Correct. It is redundant there. Didn't affect css output though.

**Comment by wilsonge** [comment]
Thankyou :)

**Comment by spignataro** [comment]
Thank you peter.

**Comment by nonumber** [comment]
Great, thanks :)

**Pull request merged**